src/bins/driver.rs

```diff
@@ -56,17 +69,24 @@ fn compile_time_sysroot() -> Option<String> {
     }
 }
 
+// We create an empty struct on which we will implement the callbacks,
+// that the compiler should run for our analysis to be conducted.
```

Comment:
This is the type definition, not the instantiation

Reply:
Sure, det er bare blevet skrevet på en dum måde :)
Se min anden kommentar for et ændringsforslag.